src/open_inwoner/templates/cms/plugins/userfeed/userfeed.html

```diff
@@ -16,35 +16,21 @@ <h2 class="h2 {% if userfeed.action_required %}indicator{% endif %}">
     </div>
     <div class="card-container card-container--columns-2">
         {% for item in userfeed.items %}
-        <a href="{{ item.action_url }}" class="card card--status card--status-{{ item.status_indicator }} {% if item.is_completed %}card--completed{% endif %}">
-
-            <div class="userfeed-marker userfeed-marker--{{ item.status_indicator }}">
-                {% if item.status_indicator == "info" %}
-                    {% icon icon="info" outlined=True %}
-                {% elif item.status_indicator == "success" %}
-                    {% icon icon="check_circle" outlined=True %}
-                {% elif item.status_indicator == "failure" %}
-                    {% icon icon="cancel" outlined=True %}
-                {% elif item.status_indicator == "warning" %}
-                    {% icon icon="warning_amber" outlined=True %}
-                {% else %}
-                    {% icon icon="info" outlined=True %}
+            <a href="{{ item.action_url }}" class="card card--status card--status--{{ item.status_indicator }} {% if item.is_completed %}card--completed{% endif %}">
+                {% if item.status_text %}
+                    {% include "components/StatusIndicator/StatusIndicator.html" with status_indicator=item.status_indicator status_indicator_text=item.status_indicator_text %}
```

Comment:
This needs to be `status_text` to make sure it uses the right property: https://github.com/maykinmedia/open-inwoner/blob/develop/src/open_inwoner/userfeed/hooks/case_status.py#L106

```suggestion
                    {% include "components/StatusIndicator/StatusIndicator.html" with status_indicator=item.status_indicator status_indicator_text=item.status_text %}
```